config/vm.args

```diff
@@ -1,4 +1,4 @@
--sname aeternity@localhost
+-sname epoch@localhost
```

Comment:
[Docker config](https://github.com/aeternity/aeternity/blob/6248401116910cf053d4d184cb723b6c544ddd0d/docs/docker.md#node-configuration) is already correct.